zapcli/cli.py

```diff
@@ -430,6 +430,54 @@ def report(zap_helper, output, output_format):
     console.info('Report saved to "{0}"'.format(output))
 
 
+@cli.command('test-alerts')
+@click.option('--config-file', '-c', default=None, type=str,
+              help='config file to use to INFO, IGNORE or FAIL warnings')
+@click.option('--config-url', '-u', default=None, type=str,
+              help='URL of config file to use to INFO, IGNORE or FAIL warnings')
+@click.option('--generate', '-g', is_flag=True, default=False,
+              help='generate default config file (all rules set to WARN)')
+@click.option('--info', '-i', is_flag=True, default=False,
+              help='default rules not in the config file to INFO')
+@click.option('--level', '-l', default='PASS', type=str,
+              help='minimum level to show: PASS, IGNORE, INFO, WARN or FAIL,' +
+              'use with -s to hide example URLs. Default: PASS')
+@click.option('--quiet', '-q', is_flag=True, default=False, type=str,
+              help='short output format - dont show PASSes or example URLs')
+@click.pass_obj
+def test_alerts(zap_helper, **options):
+    """Test zap alert and print the results, according to the configuration"""
```

Comment:
I'm not sure "Test zap alert" makes sense as a help message (this doc is displayed when you pass `--help`) to explain the command, particularly as to why a user may want to use this instead of the other `alerts` command (which can also output an exit code and you configure which alerts to enable and so forth). I'm not sure of what the best wording would be here, maybe something referencing the baseline script this is compatible with?